tap_pendo/streams.py

```diff
@@ -389,15 +388,22 @@ def sync_substream(self, state, parent, sub_stream, parent_response):
 
         # Slice response for >= last processed
         if last_processed:
-            for i, e in enumerate(parent_response):
+            ## TODO: What happens when there's a list that comes in here????
+            for e in parent_response:
```

Comment:
Can we please not name this e? Single letters I think are ok in single-line comprehensions, but outside of that I strongly prefer clearer variable names.